src/ExportProcessors.mss

```diff
@@ -297,16 +297,21 @@ function ProcessLyric (lyricobj, objectPositions) {
         sylel_last = sylel;
         libmei.AddChild(verse, sylel);
 
-        // Split syllable text by underscore, in case there is an elision
-        syllables = MSplitString(syl.Text, '_');
+        // Split lyrics text by underscore and space, in case there is an elision
+        syllables = SplitStringIncludeDelimiters(syl.Text, '_ ');
         libmei.SetText(sylel, syllables[0]);
 
         // Handle any found elisions
-        for s = 1 to syllables.Length
+        for s = 1 to syllables.Length step 2
         {
-            libmei.AddAttribute(sylel_last, 'con', 'b');
+            elisionDelimiter = syllables[s];
+            elisionSyl = syllables[s + 1];
+            if (elisionDelimiter = '_')
```

Comment:
If there is a space between syllables it is preferred to also encode this with `@con="s"`, see https://music-encoding.org/guidelines/v4/attribute-classes/att.syl.log.html